client/app/components/calendar/CalendarHeader.jsx

```diff
@@ -8,7 +8,7 @@ const CalendarHeader = (props) => (
 );
 
 CalendarHeader.propTypes = {
-  day:        React.PropTypes.string.isRequired,
+  day:        React.PropTypes.oneOfType([React.PropTypes.instanceOf(Date), React.PropTypes.string]).isRequired,
```

Comment:
Can we just inject one of these types, eg. Date? It's always confusing when you have to handle more than one type